src/natcap/invest/ndr/ndr_core.pyx

```diff
@@ -115,49 +115,30 @@ def ndr_eff_calculation(
 
     for offset_dict in pygeoprocessing.iterblocks(
             (mfd_flow_direction_path, 1), offset_only=True, largest_block=0):
-        win_xsize = offset_dict['win_xsize']
-        win_ysize = offset_dict['win_ysize']
-        xoff = offset_dict['xoff']
-        yoff = offset_dict['yoff']
-        for row_index in range(win_ysize):
-            global_row = yoff + row_index
-            for col_index in range(win_xsize):
-                global_col = xoff + col_index
+        for row_index in range(offset_dict['win_ysize']):
+            global_row = offset_dict['yoff'] + row_index
+            for col_index in range(offset_dict['win_xsize']):
+                global_col = offset_dict['xoff'] + col_index
```

Comment:
Sure, I put them back and added a comment explaining it, since usually we avoid creating new variables for dictionary values like that. Unfortunately, casting to int like `<int>offset_dict['win_xsize']` doesn't seem to remove the python overhead.

Reply:
>Unfortunately, casting to int like `<int>offset_dict['win_xsize']` doesn't seem to remove the python overhead.

Yeah, that makes sense ... I am not aware of a way to statically type dictionary accesses, since the values could be any type.